x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -226,8 +226,9 @@ const NetworkDetailsLinkComponent: React.FC<{
     [flowTarget, navigateToApp, search]
   );
   const getHref = useCallback(
-    (cIp: string) => formatUrl(getNetworkDetailsUrl(encodeURIComponent(encodeIpv6(cIp)))),
-    [formatUrl]
+    (cIp: string) =>
+      formatUrl(getNetworkDetailsUrl(encodeURIComponent(encodeIpv6(cIp)), flowTarget, search)),
```

Comment:
NIT: we could use the new way to create links here, it would sipmlify the component implementation:
```
const getSecuritySolutionProps = useGetSecuritySolutionLinkProps();

const getLink = useCallback(
  (cIp: string, i: number) => {
    const { onClick, href } = getSecuritySolutionProps({
      deepLinkId: SecurityPageName.network,
      path: getNetworkDetailsUrl(encodeURIComponent(encodeIpv6(cIp)), flowTarget),
    });
    [...]
```

Reply:
nice, will make the change.